content/agent/kubernetes/container_installation.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Kubernetes container installation
+kind: documentation
+---
+
+## Setup
+
+Take advantage of DaemonSets to automatically deploy the Datadog Agent on all your nodes (or on specific nodes by using nodeSelectors).
```

Comment:
Can we link to https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector ?

Reply:
👌